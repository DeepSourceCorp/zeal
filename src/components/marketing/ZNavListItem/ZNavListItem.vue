<template>
  <li class="flex items-center px-4 py-2 cursor-pointer hover:text-juniper">
    <slot></slot>
  </li>
</template>

<script lang="ts">
import Vue from 'vue'

export default Vue.extend({
  name: 'ZNavListItem'
})
</script>
