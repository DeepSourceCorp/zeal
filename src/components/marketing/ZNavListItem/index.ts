import ZNavListItem from './ZNavListItem.vue'

export default ZNavListItem
