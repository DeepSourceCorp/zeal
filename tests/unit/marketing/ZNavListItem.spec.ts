import Vue from 'vue'
import { createLocalVue, mount, Wrapper } from '@vue/test-utils'

import ZNavListItem from '../../../src/components/marketing/ZNavListItem'

describe('ZNavListItem component', () => {
  let mountFunction: (options?: Record<string, unknown>) => Wrapper<Vue>
  let localVue: typeof Vue
  beforeEach(() => {
    localVue = createLocalVue()
    mountFunction = (options = {}) => {
      return mount(ZNavListItem, {
        localVue,
        ...options
      })
    }
  })
  it('renders a ZNavListItem when the component is mounted', () => {
    expect(mountFunction().html()).toMatchSnapshot()
  })
})
