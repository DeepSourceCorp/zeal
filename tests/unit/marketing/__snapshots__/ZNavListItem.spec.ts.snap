// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZNavListItem component renders a ZNavListItem when the component is mounted 1`] = `<li class="flex items-center px-4 py-2 cursor-pointer hover:text-juniper"></li>`;
